models/endpoints.models.js

```diff
@@ -0,0 +1,5 @@
+const endpoints = require("../endpoints.json");
+
+exports.fetchEndpoints = async () => {
+   return endpoints;
```

Comment:
I don't think you need this logic to be in a model because it isn't accessing the database, this is just requiring in a file which is done synchronously